resources/views/site/blocks/feature_block.blade.php

```diff
@@ -112,12 +110,20 @@
                                             'xlarge' => 38,
                                         ))))
                                 @endcomponent
+                                @if ($item->type === 'exhibition' && $item->is_now_open || $item->is_closing_soon || $item->is_ongoing)
+                                <div class="m-feature-block-listing__label__overlay">
+                                    <span class="{{$item->is_now_open ? 'label-open' : ''}} {{$item->is_closing_soon ? 'label-closing' : ''}}">
+                                        {{$item->is_now_open ? 'Now Open' : ''}}
+                                        {{$item->is_closing_soon ? 'Closing Soon' : ''}}
+                                    </span>
+                                </div>
+                                @endif
                             @else
                                 <span class='default-img m-feature-block-listing__img {{$ratio}} column-{{$columns}}x'></span>
                             @endif
                         </div>
-                        <span class="m-feature-block-listing__meta">
-                            @if ($item->type && $item->type !== 'experience')
+                        <span class="m-feature-block-listing__meta {{$item->type ? "l-type-$item->type" : ""}}">
+                            @if ($item->type && $item->type !== 'experience' && $item->type !== 'exhibition' && $item->type !== 'event')
```

Comment:
The output inside this `@if` seems like it would apply to exhibitions and events. Can you explain why you're checking if the type is _not_ those things?